compiler-cli/src/http.rs

```diff
@@ -42,3 +43,21 @@ impl gleam_core::io::HttpClient for HttpClient {
             .map_err(Error::http)
     }
 }
+
+fn init_client() -> Client {
+    match get_certificate() {
+        Ok(cert) => Client::builder()
+            .add_root_certificate(cert)
+            .build()
+            .expect("Unable to build reqwest client with certificate"),
+        _ => Client::new(),
+    }
+}
+
+fn get_certificate() -> Result<Certificate, Box<dyn std::error::Error>> {
+    let certificate_path = std::env::var("GLEAM_CACERTS_PATH")?;
+    let certificate_bytes = std::fs::read(&certificate_path)?;
+    let certificate = Certificate::from_pem(&certificate_bytes)?;
+
+    Ok(certificate)
+}
```

Comment:
In the event that the certs paths has been set we want to error if the certs cannot be read or parsed, rather than silently discarded as they are here.

Reply:
I don't think this is usual behaviour.. It's quite common for runtimes like Python and Node to ignore missing certs as it is common behaviour to add the environment variable to your shell and not populate the certificate path until necessary.